test/emf/emf_test.go

```diff
@@ -56,13 +57,38 @@ func getEcsTestRunners(env *environment.MetaData) []*test_runner.ECSTestRunner {
 	return ecsTestRunners
 }
 
+func getEksTestRunners(env *environment.MetaData) []*test_runner.EKSTestRunner {
+	if eksTestRunners == nil {
+		factory := dimension.GetDimensionFactory(*env)
+
+		eksTestRunners = []*test_runner.EKSTestRunner{
+			{
+				Runner: &EMFTestRunner{test_runner.BaseTestRunner{DimensionFactory: factory}, "EMF_EKS"},
```

Comment:
If the only difference is the namespace here, maybe instead of explicitly passing it in as such, it could be derivable from the `env`? I think it would be nice to be able to consolidate so we don't have separate `get X runners` functions that create structs with the same things in them. Not super pressing to do now.